src/pages/likes/Likes.jsx

```diff
@@ -14,7 +14,7 @@ export function Likes() {
       <div className="sm-main-feature-p">
         <h2 className="prod-heading">Liked Videos</h2>
         <div className="sm-main-prod-container">
-          {likes.length === 0 ? (
+          {likes.length === 0 && (
             <Link to="/" title="Explore Videos">
               <button
```

Comment:
Suggestion: It is better to use ternary instead of short-circuiting in JSX. As while short-circuiting if there is a string JSX will render it in the view.

For more info: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx